src/lib/knex.js

```diff
@@ -1,25 +1,17 @@
 'use strict'
 
-function getKnexConfigType () {
-  if (process.env.LEDGER_DB_ENV) {
-    return process.env.LEDGER_DB_ENV
-  }
-  if (process.env.LEDGER_UNIT_DB_URI) {
-    if (process.env.LEDGER_UNIT_DB_URI.startsWith('mysql:')) {
-      return 'mysql'
-    }
-    if (process.env.LEDGER_UNIT_DB_URI.startsWith('postgres:')) {
-      return 'postgres'
-    }
-  }
-  return 'development'
-}
+const config = require('../services/config')
 
```

Comment:
I think this isn't being done correctly for mysql and postgres. I'll try to think of a good solution.